types/office-js/index.d.ts

```diff
@@ -29867,27 +30124,35 @@ declare namespace Excel {
      * Represents the value of a cell containing a number with a format string. Number format strings must conform to Excel guidelines. To learn more, see {@link https://support.microsoft.com/office/c0a1d1fa-d3f4-4018-96b7-9c9354dd99f5  | Review guidelines for customizing a number format}.
      * In this scenario, the format is applied to the value and not to the cell, so the value retains its format string throughout calculation.
      *
+     * @deprecated since [Api set: ExcelApi 1.19]. Use {@link https://learn.microsoft.com/javascript/api/excel/excel.doublecellvalue | DoubleCellValue} instead.
+     *
      * @remarks
      * [Api set: ExcelApi 1.16]
      */
     interface FormattedNumberCellValue {
         /**
          * Represents the type of this cell value.
          *
+         * @deprecated since [Api set: ExcelApi 1.19].
```

Comment:
```suggestion
         * @deprecated Deprecated since [Api set: ExcelApi 1.19].
```

Reply:
Or use the same message as the interface.

Reply:
Here and other instances

Reply:
Thank you! Added this throughout